### PR TITLE
Peter telemetry dev

### PR DESCRIPTION
**Pull request by rydzynskip**
Created Platformio projects for each of the Teensy CAN nodes, along with initial script for telemetry data handling

### CODE REVIEW

File: ArduinoCode/Dashboard/Dashboard.ino [hunk 1]
```diff
@@ -3,6 +3,16 @@
 
 #include <mcp_can.h> // uses seeed-studio's CAN_BUS_Shield library
 
+
+#define CAN_FAULT_LED      0
+#define GENERAL_LED_2      1 // custom behavior
```
Comment (mccauleyma) [inline ArduinoCode/Dashboard/Dashboard.ino:8]: Is LED2 = 1 and LED1 = 2 based on schematics? Cause for the sake of satisfaction we can definitely switch it

File: Embedded Software/Dashboard/src/main.cpp [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+#define MAX_MB_NUM 16 // maximum number of CAN mailboxes to use 
+
+// CAN IDs
+#define MC_STATES_ID       0xAA
+#define CAN_START_MOTOR_ID 0x101
+#define CAN_DIRECTION_ID   0x102
+
+// pin values
+#define CAN_FAULT_LED      0
+#define GENERAL_LED_2      1 // custom behavior
```
Comment (mccauleyma) [inline Embedded Software/Dashboard/src/main.cpp:14]: Same comment as Arduino Code. Is 1 v 2 for a reason?

**Review by mccauleyma — CHANGES_REQUESTED** [review]
Overall looks good. Just one random comment on pin numbering

**Review by mccauleyma — APPROVED** [review]
Yep. We can deal with schematics integration later

**Review by nwdepatie — APPROVED** [review]
We good, I think we can change it as needed during testing and integration

**Pull request merged**